src/plugins/intel_cpu/src/transformations/snippets/x64/pass/eliminate_brgemm_copy_b.hpp

```diff
@@ -16,10 +17,22 @@ namespace ov::intel_cpu::pass {
  *
  * @ingroup snippets
  */
-class EliminateBrgemmCopyB : public ov::pass::MatcherPass {
+class EliminateBrgemmCopyB : public ov::pass::ModelPass {
 public:
-    OPENVINO_MATCHER_PASS_RTTI("EliminateBrgemmCopyB");
-    EliminateBrgemmCopyB();
+    OPENVINO_MODEL_PASS_RTTI("EliminateBrgemmCopyB");
+    EliminateBrgemmCopyB(std::set<size_t> constant_inputs_idxs,
+                         ov::intel_cpu::RepackedInputConfig& repacked_runtime_inputs_config,
+                         ov::intel_cpu::RepackedInputConfig& repacked_constant_inputs_config)
+        : m_constant_inputs_idxs(std::move(constant_inputs_idxs)),
+          m_repacked_runtime_inputs_config(repacked_runtime_inputs_config),
+          m_repacked_constant_inputs_config(repacked_constant_inputs_config) {}
+
+    bool run_on_model(const std::shared_ptr<ov::Model>& model) override;
+
+private:
+    std::set<size_t> m_constant_inputs_idxs;
```

Comment:
It seems like we mustn't change this set in the pass
```suggestion
    const std::set<size_t> m_constant_inputs_idxs;
```

Reply:
Great catch! Done, thanks